src/components/Card/Card.tsx

```diff
@@ -1,58 +1,215 @@
+  /** Optional value to control the aspect ratio of the cartd image; default value is `square` */
+  imageAspectRatio?: CardImageRatios;
+  /** Optional value to control how the image is sized image */
+  imageSize?: CardImageSizes;
+  /** The src attribute is required, and contains the path to the image you want to embed. */
+  src: string;
+}
+
+interface CardHeadingProps {
+  /** Optional className that appears in addition to `heading` */
+  className?: string;
+  /** Optional size used to override the default styles of the semantic HTML `<h>` elements */
+  displaySize?: HeadingDisplaySizes;
+  /** Optional ID that other components can cross reference for accessibility purposes */
+  id?: string;
+  /** Number 1-6; used to create the `<h*>` tag */
+  level: number;
+  /** Optional URL that header points to; when `url` prop is passed to `Heading`, a child `<a>` element is created and the heading text becomes an active link */
+  url?: string;
+}
+
+interface CardContentProps {
+  /** Optional className that would be applied to the `card-content` element */
+  className?: string;
+}
+
+interface CardActionsProps {
+  /** Optional boolean value to control visibility of border on the bottom edge of the card actions element */
+  bottomBorder?: boolean;
+  /** Optional className that would be applied to the `card-actions` element */
   className?: string;
+  /** Modifiers array for use with BEM. See how to work with modifiers and BEM here: http://getbem.com/introduction/ */
+  modifiers?: string[];
+  /** Optional boolean value to control visibility of border on the top edge of the card actions element */
+  topBorder?: boolean;
+}
+
+export interface CardProps {
+  /** Optional hex color value used to set the card background color */
+  backgroundColor?: string;
   /** BlockName for use with BEM. See how to work with blockNames and BEM here: http://getbem.com/introduction/ */
   blockName?: string;
-  /** Calls to Action for the card, such as "download" on a card for a book */
-  ctas?: React.ReactNode;
-  /** Footer that appears within the card but below the three columns within it */
-  footer?: React.ReactNode;
-  /** Heading that appears within the card but above the three columns within it */
-  heading?: React.ReactNode;
+  /** Optional boolean value to control the visibility of a border around the card */
+  border?: boolean;
+  /** Optional boolean value to control the alignment of the text and elements within the card */
+  center?: boolean;
+  /** ClassName that appears in addition to "card" */
+  className?: string;
+  /** Optional hex color value used to override the default text color */
+  foregroundColor?: string;
   /** ID that other components can cross reference for accessibility purposes */
   id?: string;
-  /** First column within the card */
-  image?: React.ReactNode;
+  /** Optional value to control the aspect ratio of the cartd image; default value is `square` */
+  // imageAspectRatio?: CardImageRatios;
```

Comment:
Remove this and `imageSize`.

Reply:
These both came back into play, so they are still in there and uncommented.